lib/rucio/web/rest/webpy/v1/redirect.py

```diff
@@ -142,17 +143,7 @@ def GET(self, scope, name):
                 yield '  <glfn name="/atlas/rucio/%s:%s">' % (rfile['scope'], rfile['name'])
                 yield '</glfn>\n'
 
-                # sort the actual replicas if necessary
-                if select == 'geoip':
-                    replicas = sort_geoip(dictreplica, client_location['ip'], ignore_error=True)
-                elif select == 'closeness':
-                    replicas = sort_closeness(dictreplica, client_location)
-                elif select == 'dynamic':
-                    replicas = sort_dynamic(dictreplica, client_location)
-                elif select == 'ranking':
-                    replicas = sort_ranking(dictreplica, client_location)
-                else:
-                    replicas = sort_random(dictreplica)
+                replicas = sort_replicas(dictreplica, client_location, selection=select, default='random')
```

Comment:
should this be default=geoip instead?

Reply:
no, the redirect endpoint is out of scope.